test/test_hpyerr.py

```diff
@@ -15,3 +15,16 @@ def test_NoMemory(self):
         """)
         with pytest.raises(MemoryError):
             mod.f()
+
+    def test_FatalError(self):
+        mod = self.make_module("""
+            HPyDef_METH(f, "f", f_impl, HPyFunc_NOARGS)
+            static HPy f_impl(HPyContext ctx, HPy self)
+            {
+                HPy_FatalError(ctx, "boom!");
+            }
+            @EXPORT(f)
+            @INIT
+        """)
+        # Calling mod.f() gives a fatal error, ending in abort().
+        # How to check that?  For now we just check that the above compiles
```

Comment:
I fear the only solution is to write some code in a tmp file, spawn a subprocess and check the output/return code. It will be messy to test this on PyPy, though.

Reply:
Yes, and I think for now it's fine like it is